src/main/scala/cromwell/engine/backend/Backend.scala

```diff
@@ -37,6 +37,7 @@ object Backend {
  * Trait to be implemented by concrete backends.
  */
 trait Backend {
+  type A <: BackendCall
```

Comment:
Deleted the previous comment.

I'd still argue that it's more idiomatic to use single letters, although I had stated that single letters started from A is suggested by the style guide. However the style guide makes a distinction between "simple type parameters" (A, B, C) and otherwise (full names) and their examples make it clear that this is not the former - despite their own official code using letters in situations that their example gives words.

I'd argue that `BC` is the worst of both worlds. It doesn't add full meaning and it's longer than `A`.

I think that full word abstract type parameters are stupid as hell but our shiny new style guide says that the official style guide is the base case, so it is what it is. It should be `BackendCall` according to the guide (http://docs.scala-lang.org/style/naming-conventions.html#type_parameters_generics)